pkgs/applications/misc/rtfm/gtk-docs.nix

```diff
@@ -0,0 +1,77 @@
+{ stdenv
+, fetchFromGitHub
+, gi-docgen
+, gobject-introspection
+, libadwaita
+, gtk4
+, pango
+}:
+stdenv.mkDerivation (finalAttrs: {
+  name = "gtk-docs";
+
+  adw1_version = "1.3.3";
+  adw1_docs = fetchFromGitHub {
+    owner = "GNOME";
+    repo = "libadwaita";
+    rev = "refs/tags/${finalAttrs.adw1_version}";
+    hash = "sha256-YIxGwl+/F7xkGjoi07GViSHAfCTE1RpEBhHfrlD0X/4=";
+  };
+
+  gtk4_version = "4.11.4";
+  gtk4_docs = fetchFromGitHub {
+    owner = "GNOME";
+    repo = "gtk";
+    rev = "refs/tags/${finalAttrs.gtk4_version}";
+    hash = "sha256-YobWcLJm8owjrz6c6aPMCrVZqYDvNpjIt5Zea2CtAZY=";
+  };
+
+  pango_version = "1.50.14";
+  pango_docs = fetchFromGitHub {
+    owner = "GNOME";
+    repo = "pango";
+    rev = "refs/tags/${finalAttrs.pango_version}";
+    hash = "sha256-b3N9kM8xZ/FkpPR9gYJz96tiG/XUOUB6ipfLmBllCKc=";
+  };
```

Comment:
Do we really must pin those 3?

Reply:
I think so, because they are marked as dependencies on the project's github, that is, the author assumes that during the process of building the program, all the documentation is present in the system.
```md
- Offline crystal api documentation for Crystal docset generation.
- Offline Gtk api documentation for Gtk4, Gdk4, Gsk4, libAdwaita and Pango libraries for GTK4 libraries docset generation.
```

We can build a program without documentation at all, but in this case `rtfm` will be useless. It also makes no sense to set part of the documentation, since in this case the user himself will have to generate docsets from sources.

Reply:
I didn't find docs in gtk/pango/libadwaita nix packages.